modules/logging-loki-gui-install.adoc

```diff
@@ -35,14 +33,132 @@ include::snippets/logging-stable-updates-snip.adoc[]
 +
 The {loki-op} must be deployed to the global operator group namespace `openshift-operators-redhat`, so the *Installation mode* and *Installed Namespace* are already selected. If this namespace does not already exist, it is created for you.
 
-. Select *Enable operator-recommended cluster monitoring on this namespace.*
+. Select *Enable Operator-recommended cluster monitoring on this namespace.*
 +
 This option sets the `openshift.io/cluster-monitoring: "true"` label in the `Namespace` object. You must select this option to ensure that cluster monitoring scrapes the `openshift-operators-redhat` namespace.
 
 . For *Update approval* select *Automatic*, then click *Install*.
 +
 If the approval strategy in the subscription is set to *Automatic*, the update process initiates as soon as a new Operator version is available in the selected channel. If the approval strategy is set to *Manual*, you must manually approve pending updates.
 
+. Install the Red{nbsp}Hat OpenShift Logging Operator:
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Choose  *Red{nbsp}Hat OpenShift Logging* from the list of available Operators, and click *Install*.
+
+.. Ensure that the *A specific namespace on the cluster* is selected under *Installation Mode*.
+
+.. Ensure that *Operator recommended namespace* is *openshift-logging* under *Installed Namespace*.
+
+.. Select *Enable Operator recommended cluster monitoring on this namespace*.
++
+This option sets the `openshift.io/cluster-monitoring: "true"` label in the Namespace object.
+You must select this option to ensure that cluster monitoring
+scrapes the `openshift-logging` namespace.
+
+.. Select *stable-5.y* as the *Update Channel*.
+
+.. Select an *Approval Strategy*.
++
+* The *Automatic* strategy allows Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available.
++
+* The *Manual* strategy requires a user with appropriate credentials to approve the Operator update.
+
+.. Click *Install*.
+
+. Go to the *Operators* -> *Installed Operators* page. Click the *All instances* tab.
+
+. From the *Create new* drop-down list, select *LokiStack*.
+
+. Select *YAML view*, and then use the following template to create a `LokiStack` CR:
++
+--
+.Example `LokiStack` CR
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki # <1>
+  namespace: openshift-logging # <2>
+spec:
+  size: 1x.small # <3>
+  storage:
+    schemas:
+    - version: v12
```

Comment:
Review the comment before for the schema version where it could be not be v12 as documented in https://issues.redhat.com/browse/OBSDOCS-787 and https://access.redhat.com/solutions/7063482. 

Starting in Logging 5.9 the version should be v13 or an alert is triggered